ingest.py

```
@@ -0,0 +1,205 @@
+            dest_file_path.parent.mkdir(parents=True, exist_ok=True)
+            
+            count += 1
+            print(f"Processing: {rel_path} -> {dest_file_path}")
+            
+            # 3. Transform & Write
+            final_content = transform_content(file_path, dest_file_path)
+            
+            with open(dest_file_path, 'w', encoding='utf-8') as f:
+                f.write(final_content)
+                
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)